extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php

```diff
@@ -39,8 +39,6 @@ public function getTileSets() {
 	 * @throws InvalidParameterApiException
 	 */
 	public function createMap() {
-		$type = trim( $this->request->getVal( 'type', WikiaMaps::MAP_TYPE_GEO ) );
-
 		$this->setData( 'tileSetId', $this->request->getInt( 'tileSetId', 0 ) );
 		$this->setData( 'title', trim( $this->request->getVal( 'map-title', '' ) ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)